### PR TITLE
Add Oceanhack setup intro and conda tutorial

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR adds initial setup introduction, which the content is borrowed from geohackweek. This also applies for the short conda installation tutorial.

### CODE REVIEW

**Comment by amanda-tan** [comment]
Few things: 

1. Should they be installing miniconda or Anaconda? 
2. Also, the Linux installation instructions should be changed from OSX to Linux (bullet point #2).

**Comment by valentina-s** [comment]
I think for their local install it is better if they get Anaconda: if it is their first dive with Python, most probably it will be easier to have many packages available?

File: _episodes/00-oceanhack-intro.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+keypoints:
+- "participants will conduct tutorial exercises in a cloud environment"
+- "everyone is encouraged to arrive with Python installed on their laptop for the project work"
+- "Conda package and environment manager will be used to install python and other libraries"
+
+---
+This preliminary tutorial is one that we would like you to complete before arriving at Oceanhackweek. The purpose is to learn about the minimum system requirement for the Oceanhackweek, install the necessary software used during the week, and hopefully answer any other technical question about your computer setup. Our goal is to have everyone up-and-running prior to the event so we can focus our time more productively when you arrive in person.
+
+> ## Python Version
+> We will be using **Python 3.6** only during the hackweek. We encouraged participants who comes from Python 2.7 to start reviewing the new syntax of Python 3.6.
```
Comment (valentina-s) [inline _episodes/00-oceanhack-intro.md]: encourage participants who come ....

**Comment by lsetiawan** [comment]
> I think for their local install it is better if they get Anaconda: if it is their first dive with Python, most probably it will be easier to have many packages available?

@valentina-s, previously in geohackweek, we gave them option to install Anaconda or Miniconda, should we do same here or just do Anaconda. Problem with Anaconda is that the distribution is very bloated.. Maybe we can have an instruction to install Miniconda, then install some environment file that we specified with the packages for the hackweek?

**Comment by lsetiawan** [comment]
> Also, the Linux installation instructions should be changed from OSX to Linux (bullet point #2).

Haha... will do... my bad xD

File: _episodes/01-conda-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+keypoints:
+- conda can be installed in two ways (Anaconda and Miniconda)
+- conda package manager works across systems
+
+---
+
+### What is Conda?
+[**Conda**](http://conda.pydata.org/docs/) is an **open source `package` and `environment` management system for any programming languages, but very popular among python community,** for installing multiple versions of software packages, their dependencies and switching easily between them. It works on Linux, OS X and Windows.
+
+[This Continuum blog post is a terrific, recent and comprehensive introduction to conda](http://www.continuum.io/blog/conda-data-science) targeted to data scientists. It also has links to a presentation (Youtube and slides) on the same material. An extra nifty aspect of this material is that it "explores how to use conda in a multi-language data science project" with an example combining Python and R libraries.
```
Comment (valentina-s) [inline _episodes/01-conda-tutorial.md]: the link does not work

File: _episodes/01-conda-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Installing Miniconda
+
+##### Windows
+Click [here](http://conda.pydata.org/miniconda.html) to download the proper installer for your Windows platform (32 or 64 bits).
+We recommend to download the Python 3 version of Miniconda. You can still create Python 2 environments using the Python 3 version of Miniconda.
+
+When installing, you will be asked if you wish to make the Anaconda Python your default Python for Windows.
+If you do not have any other installation that is a good option.  If you want to keep multiple versions of python on your machine (e.g. ESRI-supplied python, or both 32 and 64 bit versions of Anaconda), then don't select the option to modify your path or modify your windows registry settings.
```
Comment (valentina-s) [inline _episodes/01-conda-tutorial.md]: Do people use 32 bit versions these days? If not, maybe we can remove this example, to reduce the complexity.

File: _episodes/01-conda-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+url=bit.ly/miniconda3
+wget $url -O miniconda.sh
+bash miniconda.sh -b -p $HOME/miniconda
+export PATH="$HOME/miniconda/bin:$PATH"
+conda update conda --yes
+```
+
+### Installing Python
+We will be using Python 3.6 during the week. 
+If you are already familiar with Python 2.7, you can take a look at the syntax differences [here](http://sebastianraschka.com/Articles/2014_python_2_3_key_diff.html).
```
Comment (valentina-s) [inline _episodes/01-conda-tutorial.md]: I would add

..., but the main point to remember is to put the print statements in parentheses:

```
print("Hello World!")
```

File: _episodes/01-conda-tutorial.md (hunk 2 above)
Comment (lsetiawan) [reply _episodes/01-conda-tutorial.md]: Good catch! Hmm..  I'll remove that.

File: _episodes/01-conda-tutorial.md (hunk 4 above)
Comment (lsetiawan) [reply _episodes/01-conda-tutorial.md]: That's the most obvious difference!

**Comment by valentina-s** [comment]
👍

I don't have rights to merge, but once you do, I will add a  few more details regarding bash.

**Pull request merged**